code/modules/antagonists/eldritch_cult/eldritch_magic.dm

```diff
@@ -0,0 +1,458 @@
+/obj/effect/proc_holder/spell/targeted/ethereal_jaunt/shift/ash
+	name = "Ashen passage"
+	desc = "Low range spell allowing you to pass through a few walls."
+	school = "transmutation"
+	invocation = "ASH'N P'SSG'"
+	invocation_type = "whisper"
+	charge_max = 150
+	range = -1
+	action_icon = 'icons/mob/actions/actions_ecult.dmi'
+	action_icon_state = "ash_shift"
+	action_background_icon_state = "bg_ecult"
+	jaunt_in_time = 13
+	jaunt_duration = 10
```

Comment:
Does this even last long enough to move through a wall?

Reply:
it allows you to move thought 5 wall-ish. Remember this is one of the first spells you pick up and it is on 15 second cooldown